modules/oadp-release-notes-1-2-0.adoc

```diff
@@ -0,0 +1,90 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_content-type: REFERENCE
+[id="migration-oadp-release-notes-1-2-0_{context}"]
+= OADP 1.2.0 release notes
+
+The OADP 1.2.0 release notes include new features and lists of fixed bugs and known issues.
+
+[id="new-features_{context}"]
+== New features
+
+.Resource timeouts
+The new `resourceTimeout` option specifies how many minutes to wait for several Velero resources before timeout. This applies for such resources as Velero CRD availability, `volumeSnapshot` deletion, and backup repository availablity. 10 minutes is the default setting.
+
+.AWS S3 compatible backup storage providers
+You can back up objects and snapshots on AWS S3 compatible providers.
+
+[id="new-features-tech-preview-1-2-0_{context}"]
+=== Technical preview features
+
+.Data Mover
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
```

Comment:
@HagayVider1 - do you think we should have links to the documentation of the new feature? what do you think?

Reply:
I think we should include links for every new feature. Can you please point me in the direction where these features are documented? Tickets, PRs?